src/common/components/status-tag/status-tag.component.tsx

```diff
@@ -35,17 +45,30 @@ export class StatusTag extends Component<StatusTagProps> {
         }
     };
 
+    mapSizeToStyle = (size: StatusTagSize): CSSProperties => {
+        switch (size) {
```

Comment:
I'm a fan of this component but I think having 3 sizes is a bit convoluted, I think keeping it simple is preferrable for now

Reply:
I made two, and then I figured someone would complain about a middleground. So I added a third one. Don't really see a problem